backend/src/nodes/impl/upscale/auto_split.py

```diff
@@ -69,10 +71,9 @@ def _exact_split(
     def no_split_upscale(i: np.ndarray, r: Region) -> np.ndarray:
         result = upscale(i, r)
         if isinstance(result, Split):
-            raise _SplitEx
+            raise _SplitExError
         return result
 
-    MAX_ITER = 20
     for _ in range(MAX_ITER):
```

Comment:
Not a good change. `MAX_ITER` is specific to this one loop, so it should be as close as possible to that loop. I only used the naming conventions for constants, because there is no `const` in python, so there was no way for me to mark this variable as immutable.